#!/usr/bin/python
PACKAGE = "wheel_leg_fsm"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Lpf_param", double_t, 0, "rate lpf param", 0.4, 0.1, 0.9)
gen.add("Kalman_Q", double_t, 0, "kalman Q", 0.001, 0.0005, 0.05)
gen.add("Kalman_R", double_t, 0, "kalman R", 0.70, 0.01, 1)

exit(gen.generate(PACKAGE, "wheel_leg_fsm", "Observe"))
